gnosis/safe-react
# Feature: Add ens domain to provider info

Relates to #3231

## How this PR relates to it
ENS Resolution when adding a provider

## How to test it
- Open the Safe
- Connect a wallet that has a registered ENS domain
- Observe `ensDomain` in redux `providers`

## Screenshots
![Screenshot 2022-01-11 at 12 40 32](https://user-images.githubusercontent.com/5880855/148936388-34babf40-37c9-4e33-af72-ff4cebc930a1.png)
